Add version file which if missing breaks when rebuilding the configure script.
Author: micketeer@gmail.com per request of Simon Branford
--- prov/psm3/VERSION.orig	1970-01-01 01:00:00.000000000 +0100
+++ prov/psm3/VERSION	2021-08-08 00:19:07.634693856 +0200
@@ -0,0 +1 @@
+3_1_0_0
